app/angular/src/client/preview/angular/helpers.ts

```diff
@@ -7,6 +7,12 @@ import { STORY } from './app.token';
 import { NgModuleMetadata, IStoryFn, NgStory } from './types';
 import { ReplaySubject } from 'rxjs';
 
+declare global {
```

Comment:
@ndelangen @kroeder @gaetanmaisse @leoyli is this correct? want to merge & release this but concerned about this bit

Reply:
Isn't 'string' the value "string" and not a general string type?

Reply:
I think you're right @shilman it should be `NODE_ENV: string | 'development' | undefined;`

Reply:
And this `declare` block can be moved to a `typings.d.ts` as soon as https://github.com/storybooks/storybook/pull/6570 will be merged to avoid to have it in the middle of code file.